src/commands/sourcemaps/upload.ts

```diff
@@ -37,6 +37,7 @@ export class UploadCommand extends Command {
   public static paths = [['sourcemaps', 'upload']]
 
   public static usage = Command.Usage({
+    category: 'Source Code Integration',
```

Comment:
```suggestion
    category: 'RUM,
```
Although this command is still maintained by Source Code Integration (for legacy reasons), it's related to the RUM product, would be nice to have it in this category !

Reply:
@rpelliard @lraucy should I update the `CODEOWNERS` file to:

```diff
## Source Code Integration
src/commands/git-metadata  @DataDog/source-code-integration
+ # Related to RUM, but maintained by Source Code Integration (for legacy reasons)
src/commands/sourcemaps    @DataDog/source-code-integration
```

I generally use that file as a source of truth when categorizing our changes in the release notes.

Reply:
This makes sense to me for now 👍 I'll follow-up with the RUM team to figure out if we can switch it to them eventually

Reply:
Updated it here: https://github.com/DataDog/datadog-ci/pull/1034